### DISCUSSION TITLE
Map view of all approved videos of the protesters

### DISCUSSION THREAD

**Discussion by MeysamQaderi** [WeAreMahsaAmini/Core#180] [Ideas]
یک نقشه ای باشه که همه بتونن گستردگی اعتراضات رو ببینن.
هم دنیا بدونه که این موضوع فراگیر در کل ایرانه نه فقط تهران وهمچنین مردم ببینن که هموطن هاشون چطور جاهای مختلف در حال مبارزه هستن و روحیه بگیرن.

**Comment by kaveh6** [comment]
duplicate https://github.com/WeAreMahsaAmini/Core/discussions/175
